### ISSUE TITLE
Feature request: Pokedex information on unique pokemon caught

### ISSUE THREAD

**Issue by tteltrab** [sharloteean/Hangman-Game#1]
Knowing the total number is, of course, valuable - but as a pokemon master my goal is to catch them all!

```gherkin
Feature: List of unique pokemon caught
  As a pokemon master
  I want a list of all of the unique pokemon I have caught
  So that I know when I've caught them all.
```

**Comment by tteltrab** [comment]
Let me know if you're open to PRs! I can give it a STAB.

**Comment by charmanon** [comment]
Yes, please, go crazy! I was going to add more features like being able to choose your starter and pokemon types affecting how fast health was lost. Right now the pokedex is going to be very short (I have 16 out of the 721 in the game) but I'm going to keep adding more. Feel free to do whatever you want.
